docs/_tutorials/ja_getting_started.md

```diff
@@ -161,7 +161,8 @@ const app = new App({
   token: process.env.SLACK_BOT_TOKEN,
   signingSecret: process.env.SLACK_SIGNING_SECRET,
   socketMode: true,
-  appToken: process.env.SLACK_APP_TOKEN
+  appToken: process.env.SLACK_APP_TOKEN,
+  port: process.env.PORT || 3000 // ソケットモードはポートでリッスンしませんが、アプリがOAuthに応答するようにしたい場合は、ポートでリッスンする必要があります。
```

Comment:
@filmaj Thanks for the draft (it's almost perfect 😄). The following one would be good enough:

```suggestion
  port: process.env.PORT || 3000 // ソケットモードではポートをリッスンしませんが、アプリを OAuth フローに対応させる場合、何らかのポートをリッスンする必要があります
```

if you have two lines for better readability,

```suggestion
  // ソケットモードではポートをリッスンしませんが、アプリを OAuth フローに対応させる場合、
  // 何らかのポートをリッスンする必要があります
  port: process.env.PORT || 3000
```